README.md

```diff
@@ -75,6 +75,11 @@ By starting the workflow, we understand starting all tasks having no parents (`T
   * default is `['all_succeeded', {}]` - which means that all parents have to finish with success
   * other available rule: `['number_succeeded', {number: 2}]` - at least 2 parents succedeed
 
+* `inline`
+  * Sidekiq execution may be quite slow when we have a succession of tasks (~ 5s per task).
```

Comment:
Maybe there is a problem in the sidekiq flow gem that leads to such a long exec time ?